source/guides/tooling/intelligent-code-completion.md

```diff
@@ -117,6 +117,12 @@ To set up in {% url "Visual Studio Code" https://code.visualstudio.com/ %} you c
 }
 ```
 
+Or you can directly add a `$schema` key to the cypress config, which is also a way of sharing this setting with collaborators. 
```

Comment:
When you say 'cypress config', I'm not sure where you are saying to put this - is this in the `cypress.json` file? Can you clarify - link to an external doc if possible explaining this `$schema` key. Thanks!

Reply:
Hi @jennifer-shehane 

yes, directly in the cypress.json. Example:

```json
{
  "$schema": "https://on.cypress.io/cypress.schema.json",
  ...other cypress options
}
```

If you need more detail, this is described here and generally works for all JSON files:
https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings

Reply:
Yup this works per project https://glebbahmutov.com/blog/json-schema-for-the-win/ you can also set it up for all Cypress json files like we already advise in the docs. I am all for this addition.

Reply:
Ok, great. This is fun because we are making the configuration file configurable in our next release, so I would just mention the 'configuration file'.

```suggestion
Or you can directly add a `$schema` key to your Cypress configuration file, which is a great way to share the schema with all collaborators of the project.
```